website/source/docs/platform/k8s/service-sync.html.md

```diff
@@ -67,7 +86,9 @@ sync to understand how the syncing works.
 The sync process must authenticate to both Kubernetes and Consul to read
 and write services.
 
-For Kubernetes, a valid kubeconfig file must be provided with cluster
```

Comment:
Previously it wasn't obvious that users running in Kubernetes didn't need to do anything for authentication.